interview-questions.md

```diff
@@ -8,36 +8,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: 
+To add the foreign key we simply need to add a new column in the terminal and run a migrate so it will populate in the scema. The foreign key when added will give a number value that can be associated with a specific student. The foreign key would belong to cohort. This is because the cohort model needs to have something to associate with students, so when we add a student to the cohort we can give student the foreign key and it will belong_to the main model which is Cohort. 
 
-Researched answer:
+Researched answer: To create the foreign key you need to create a new column for the database and give it the naming convention modelname_id. This needs to be equal to an integer. When determining where to put the foreign key, the rule of thumb is alway stick it with the belongs_to side. 
+answer via: LEARN syllabus
 
 2. Which RESTful routes must always be passed params? Why?
 
-Your answer:
+Your answer: The RESTful routes that always need params are show, edit. They need params because you are wanting to target a specific data entry. For example we would give both of these routes [params :id] so we can have the HTML form request for a certain pieace of data. 
 
-Researched answer:
+Researched answer: To function correctly put, post, get, and delete need params. This is the case due to each one of these actions need to aquire a specfic data entry(:id). The most base level param that needs to be used is the :id param so that each route can have access to the data you want to either read, update, or delete in some fashion. 
```

Comment:
I'd encourage you to review RESTful routes versus HTTP verbs.  The RESTful routes that must have a param are show, update, edit, and delete.  We need to know which of the items in the database we need to access in these cases.  Again, you have the right idea, just a little confusion of vocabulary.  You'll get it!